app/src/main/java/ch/epfl/favo/util/PictureUtil.java

```diff
@@ -0,0 +1,64 @@
+package ch.epfl.favo.util;
+
+import android.annotation.SuppressLint;
+import android.graphics.Bitmap;
+import android.net.Uri;
+
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.storage.FirebaseStorage;
+import com.google.firebase.storage.StorageReference;
+
+import java.io.InputStream;
+import java.util.concurrent.CompletableFuture;
+
+import ch.epfl.favo.common.DatabaseWrapper;
+
+@SuppressLint("NewApi")
+public class PictureUtil {
+
+  private static final String PICTURE_FILE_EXTENSION = ".jpeg";
+  private static final long TEN_MEGABYTES = 10 * 1024 * 1024;
+  private static final FirebaseStorage storage = FirebaseStorage.getInstance();
```

Comment:
You can checkout the DatabaseWrapperTest for ideas. 
You can mock the firebase storage object like this:
mockFirestore = Mockito.mock(FirebaseStorage.class);
Then you can inject it using the DependencyFactory. 

Then you can mock any expected behaviour when you call any of the methods of mockFirestore using 
Mockito.doReturn(expectedObject).when(mockFirestore).*anyMethodYouWantToMock*(any(*Argument*.class)). 

Here expectedObject can also be another mock created with Mockito.mock(*ExpectedObject*.class)